src/scripts/main.js

```diff
@@ -1,3 +1,49 @@
 'use strict';
 
-// write your code here
+function createResolvedDiv() {
+  const resolvedDiv = document.createElement('div');
+
+  resolvedDiv.className = 'message';
+  resolvedDiv.innerText = 'Promise was resolved!';
+
+  document.body.appendChild(resolvedDiv);
+}
+
+function createRejectedDiv(error) {
+  const rejectedDiv = document.createElement('div');
+
+  rejectedDiv.className = 'message error-message';
+  rejectedDiv.innerText = error;
+
+  document.body.appendChild(rejectedDiv);
+}
+
+const promise1 = new Promise((resolve, reject) => {
+  const logo = document.querySelector('.logo');
```

Comment:
Ensure that the '.logo' element exists in the DOM before trying to add an event listener to it. If the element is not found, `logo` will be `null`, and calling `addEventListener` on it will throw an error. Consider adding a check to handle this case.